packages/react/src/components/headerNavigationMenu/HeaderNavigationMenuContext.tsx

```diff
@@ -0,0 +1,14 @@
+import { createContext } from 'react';
+
+export type HeaderNavigationMenuContextProps = {
+  /**
```

Comment:
Should we start typing this kind of documentation as docstrings for the type instead? It could make the files more readable. Just a thought.

Reply:
What do you mean exactly? Can you give a little example?

Reply:
After giving it a try it seems that this is easier for IDEs (at least for vscode) to figure out than the way I suggested so let's stick with this way :)